src/window.rs

```diff
@@ -644,7 +644,7 @@ impl Window {
     ///
     /// - **iOS:** Can only be called on the main thread. Returns the `PhysicalSize` of the window's
     ///   [safe area] in screen space coordinates.
-    /// - **Web:** Returns the size of the canvas element.
+    /// - **Web:** Returns the size of the canvas element. Doesn't account for `transform` or `scale`.
```

Comment:
But that's sounds wrong then, because `PhysicalSize` must account for scale, as in it's the size after all the scaling.

Not sure that it should be the size after all the transform though, because transform is difficult to handle if it matters.

Reply:
Just to clarify we are talking about CSS [`scale`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale), not scale factor.

Would love to account for both, but unfortunately that's impossible because `ResizeObserver` doesn't account for it, so we can't fire a `Resized` (`Configured`) event when the `scale` changes. Same for `transform`, which is even more complicated because that can get really tough to calculate.

Additionally `transform` can lead to non square shaped canvases, which we can't really represent in Winit and I don't think we should try to.

Reply:
I added a clarification that this is about CSS and not scale factor.

Reply:
See https://github.com/w3c/csswg-drafts/issues/3554#issuecomment-494199650, which commented on this issue.

Reply:
Is it correct that you can simply pass this size to wgpu or some other rendering stuff without any adjustment and you'll get the correct rendering without rescaling by the browser?

Reply:
>Just to clarify we are talking about CSS scale, not scale factor.

Then you should explicitly state that you're talking about CSS and not `Window::scale_factor`.

Reply:
> Is it correct that you can simply pass this size to wgpu or some other rendering stuff without any adjustment and you'll get the correct rendering without rescaling by the browser?

Unless your are using these CSS properties, yes.

> Then you should explicitly state that you're talking about CSS and not `Window::scale_factor`.

I just discovered that CSS `scale` is a sub-property of `transform`, so I will remove the mention of it completely now.

Reply:
Done, I would appreciate another look.